### ISSUE TITLE
analytic11

### ISSUE THREAD

**Issue by wsqLab** [parpishro/FBC#23]
In descriptions or data sets themselves, as applicable ...

For the S version
(1) Add a reference if you found the example somewhere
(2)  Give the function used for the computer model
(3) Name columns 2 and 3.  BTW, rather than "xf" and "xs" I would just use "x" for both data sets: it's the same variable

For the F version
(1) Repeat the reference if you found the example somewhere
(2) Give the true value of the calibration parameter used
(3) Give the discrepancy function.  Along with the true value of the calibration parameter you can then define the true mean function
(4) Say something like: "Independent normal random error were added with mean 0 and standard deviation xxx."

Basically, the user will appreciate all the details of how you constructed the two data sets for the example.

Give a more descriptive name than "analytic".  I can help with that if I know the functions generating the two data sets.

Drop the "11" from the names as it relates only to the sim data.
